### PR TITLE
Patch isomorphic-ws package

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- Patch `isomorphic-ws` package for supporting browser.
- Caution: Tests break because we only consider browser compatibility!

Related issues:
- https://github.com/ainblockchain/ain-js/issues/171

### CODE REVIEW

File: src/event-manager/event-channel-client.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Ain from '../ain';
-import * as WebSocket from 'isomorphic-ws';
```
Comment (platfowner) [inline src/event-manager/event-channel-client.ts:2]: could you leave the original code with commenting out for future references?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/event-manager/event-channel-client.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/event-manager/event-channel-client.ts:2]: done!

Comment (platfowner) [reply src/event-manager/event-channel-client.ts:2]: ah.. I mean the code in event-channel-client.ts.

**Comment by jiyoung-an** [comment]
@platfowner Thank you for review! should we merge now, or keep the current PR status?

**Comment by platfowner** [comment]
> @platfowner Thank you for review! should we merge now, or keep the current PR status?

Please merge if you can so that we can move on to the next steps. @jiyoung-an 
Before that could you take a look at the latest comment (in the resolved thread)?

File: src/event-manager/event-channel-client.ts (hunk 1 above)
Comment (jiyoung-an) [reply src/event-manager/event-channel-client.ts:2]: @platfowner okay, done!

**Pull request merged**